tests/conftest.py

```diff
@@ -9,5 +12,12 @@ def handle_request_mocker(self, **kwargs):
         return kwargs
 
     mocker.patch.object(Client, 'handle_request', handle_request_mocker)
-    client = Client(user_id=1)
+    client = Client(user_id=USER_ID_FOR_TESTING)
     return client
+
+
+@pytest.fixture
+def run_config(path=RUN_CONFIG_PATH):
+    with open(path) as f:
```

Comment:
minor thing, i think `RUN_CONFIG_PATH` is not needed to be parsed as an argument since you define it as a constant